force-app/main/default/lwc/dataTableWrapper/__tests__/dataTableWrapper.test.js

```diff
@@ -41,6 +41,13 @@ describe("c/dataTableWrapper", () => {
             draftValues[0]
         );
     });
+    it("should display the lightning-datatable", () => {
+        const checksTable = component.shadowRoot.querySelector("lightning-datatable");
+
+        expect(checksTable.columns).not.toBe(null);
+        expect(checksTable.records).not.toBe(null);
```

Comment:
@a-contreras
Checking that these are not null is a good start, but we also know that they should default to an empty array. For these `@api` attributes, we want to confirm:
* That they default to `[]`
* That they can be set

So you could do something like this:

```
expect(checksTable.columns).not.toBe(null);
expect(checksTable.columns).toBe([]);

let expectedColumns = ['Name', Phone'];
checksTable.columns = expectedColumns;
expect(checksTable.columns).toBe(expectedColumns);
```

Reply:
Done!